test/api/controllers/servers.js

```diff
@@ -40,9 +40,7 @@ describe("controllers", function() {
       var testServer = {
         deployment_id: deployment_id,
         hostname: hostname,
-        ip_address: "127.0.0.1",
-        result: "success",
```

Comment:
Remove result and elapsed_seconds for the POST, so that we can test it as we're expecting to use it.